lib/stdlib/src/zip.erl

```diff
@@ -1510,7 +1510,7 @@ get_z_file(In0, Z, Input, Output, OpO, FB,
 					   CompSize, Input, Output, OpO, Z),
 			    In5 = skip_z_data_descriptor(GPFlag, Input, In4),
 			    %% TODO This should be fixed some day:
-			    %% In5 = Input({set_file_info, FileName, 
+			    %% In5 = Input({set_file_info, FileName,
```

Comment:
Nitpick: please skip this unrelated whitespace change.

Reply:
Sure